packages/components/src/components/tooltip/tooltip.ts

```diff
@@ -256,40 +253,94 @@ export default class SdTooltip extends SolidElement {
   }
 
   render() {
+    const isStart = this.placement.endsWith('-start');
+    const isEnd = this.placement.endsWith('-end');
+
+    const skiddingMap = new Map([
+      ['start', '2'],
+      ['end', '-2']
+    ]);
+
+    const isSm = this.size === 'sm';
     return html`
       <sd-popup
         part="base"
+        id="tooltip"
         exportparts="
           popup:base__popup,
           arrow:base__arrow
         "
-        class=${classMap({
-      tooltip: true,
-      'tooltip--open': this.open
-    })}
+        class=${cx('tooltip', this.open && 'tooltip--open')}
         placement=${this.placement}
-        distance=${this.distance}
-        skidding=${this.skidding}
+        distance="12"
+        skidding=${((skiddingMap.get(isStart ? 'start' : isEnd ? 'end' : 'default') || 0) as number) * (isSm ? -1 : 1)}
         strategy=${this.hoist ? 'fixed' : 'absolute'}
         flip
         shift
         arrow
+        arrow-padding="0"
+        auto-size="vertical"
```

Comment:
I think that's something from our debugging session, so I think you can remove it.